docs/user/ppl/cmd/rare.rst

```diff
@@ -26,10 +26,13 @@ rare [rare-options] <field-list> [by-clause] ``(available from 3.1.0+)``
 * rare-options: optional. options for the rare command. Supported syntax is [countfield=<string>] [showcount=<bool>].
 * showcount=<bool>: optional. whether to create a field in output that represent a count of the tuple of values. Default value is ``true``.
 * countfield=<string>: optional. the name of the field that contains count. Default value is ``'count'``.
+* usenull=<bool>: optional (since 3.4.0). whether to output the null value. The default value of ``usenull`` is determined by ``plugins.ppl.syntax.legacy.preferred``:
 
+ * When ``plugins.ppl.syntax.legacy.preferred=true``, ``usenull`` defaults to ``true``
+ * When ``plugins.ppl.syntax.legacy.preferred=false``, ``usenull`` defaults to ``false``
```

Comment:
[bullet points in blockquote](https://github.com/LantaoJin/search-plugins-sql/blob/291e3fc183871fe4879c195586b3431f844dbe32/docs/user/ppl/cmd/rare.rst), is it expected?

Reply:
Yes. Keep the format same with the [current stats.rst](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#syntax). I tried to use the new format to align with [Update PPL Command Documentation](https://github.com/opensearch-project/sql/pull/4562). But seems there are still many format problems in that PR. [ref](https://github.com/opensearch-project/sql/pull/4562#discussion_r2476160870)